roles/acme/defaults/main.yml

```diff
@@ -20,3 +20,5 @@ acme_remaining_days: "30"
 ### provider specific config
 acme_s3_config_region: "eu-west-1"
 acme_local_validation_path: "/var/www/html"
+amce_local_validation_path_file_owner: "runner"
```

Comment:
Why you set the owner and group this specific? Shouldn root be fine by default?

Reply:
Because the actual implemented test integration will fail if the owner is not "runner" and the group is not "docker".

Reply:
The tests may not fail, however probably all production deploys will fail with these defaults. We should probably default to `omit` so nothing is set.

Reply:
Thats true, i will adjust this in the afternoon.